native_client/deepspeech.cc

```diff
@@ -93,33 +470,38 @@ Model::Model(const char* aModelPath, int aNCep, int aNContext,
     status = ReadBinaryProto(Env::Default(), aModelPath, &mPriv->graph_def);
   }
   if (!status.ok()) {
-    mPriv->session->Close();
-    mPriv->session = NULL;
-    std::cerr << status.ToString() << std::endl;
+    (void)mPriv->session->Close();
```

Comment:
Why adding an extra `(void)` ? Because we don't care about the result of `Close()` ? Any good reason we would not check for it ?

Reply:
The added `(void)` is just to silence the warning, we were already doing it. I assume it's not handled because we're already in the middle of failing anyway. I'll add some checks.